templates/internet-of-things/smart-displays.html

```diff
@@ -0,0 +1,337 @@
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Case study</h4>
+        </div>
+      </div>
+      <h3 class="p-heading--4"><a href="https://ubuntu.com/engage/secure-digital-signage-screenly">Screenly innovates IoT fleet management for secure digital signage</a></h3>
+    </div>        
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Partners and players</h2>
+      <p>The <a href="https://snapcraft.io/store">Ubuntu Core app store</a> makes it easy for systems integrators and device makers to find solutions, while enabling OEMs to generate revenues from their devices over the entire lifetime of their products.</p>
+      <p ><a class="p-button js-invoke-modal" href="/internet-of-things/contact-us?product=digital-signage" data-testid="interactive-form-link" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Digital Signage Page - Partners and players', 'eventLabel' : 'Get in touch', 'eventValue' : undefined });">Get in touch</a></p>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-4 p-card">
+      <header class="no-border u-align--center u-vertically-center u-hide--small" style="min-height: 7rem;">
+        <a href="https://www.screenly.io/">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/3eb6002c-screenly+box-logo-horizontal+purple+%281%29.png",
+              alt="Screenly",
+              width="166",
+              height="80",
+              hi_def=True,
+              loading="lazy"
+            ) | safe
+          }}
+        </a>
+      </header>
+      <hr class="u-sv1 u-hide--small" />
+      <h3 class="p-card__title">Screenly</h3>
+      <p class="p-card__content">The world's most popular digital signage player for the Raspberry Pi.</p>
+      <a href="https://www.brighttalk.com/webcast/6793/206421?utm_source=ubuntuweb&utm_medium=dspage&utm_campaign=DSwebinar" title="View Screenly webinar">View webinar&nbsp;&rsaquo;</a>
+    </div>
+    <div class="col-4 p-card">
+      <header class="no-border u-align--center u-vertically-center u-hide--small" style="min-height: 7rem;">
+        <a href="https://www.4yousee.com.br/">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/391ea778-logo-4yousee.png",
+              alt="4YouSee",
+              height="65",
+              width="238",
+              hi_def=True,
+              loading="lazy"
+            ) | safe
+          }}
+        </a>
+      </header>
+      <hr class="u-sv1 u-hide--small" />
+      <h3 class="p-card__title">4YouSee</h3>
+      <p class="p-card__content">A complete digital signage solution to create, manage and play your digital content, and analyse your audience.</p>
+    </div>
+    <div class="col-4 p-card">
+      <header class="no-border u-align--center u-vertically-center u-hide--small" style="min-height: 7rem;">
```

Comment:
should we need the `u-hide--medium`class? 
The image looks a bit on medium screen 
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/57550290/159311317-b765b795-91c6-4d16-9563-8aafb464c752.png">


```suggestion
      <header class="no-border u-align--center u-vertically-center u-hide--small u-hide--medium" style="min-height: 7rem;">
```